zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -645,7 +644,8 @@ public String getXFrameOptions() {
     ZEPPELIN_NOTEBOOK_MONGO_COLLECTION("zeppelin.notebook.mongo.collection", "notes"),
     ZEPPELIN_NOTEBOOK_MONGO_URI("zeppelin.notebook.mongo.uri", "mongodb://localhost"),
     ZEPPELIN_NOTEBOOK_MONGO_AUTOIMPORT("zeppelin.notebook.mongo.autoimport", false),
-    ZEPPELIN_NOTEBOOK_STORAGE("zeppelin.notebook.storage", GitNotebookRepo.class.getName()),
+    ZEPPELIN_NOTEBOOK_STORAGE("zeppelin.notebook.storage",
+        "org.apache.zeppelin.notebook.repo.GitNotebookRepo"),
```

Comment:
This is because I move `ZeppelinConfiguration` from `zeppelin-zengine` to `zeppelin-interpreter`.  So ZeppelinConfiguration can not use `GitNotebookRepo` which is still in `zeppelin-zengine`. I think it should be fine to use String, because GitNotebookRepo is a public class which is exposed to users (user can configure it in zeppelin-site.xml), so it is expected to be an stable API.